src/Components/OpiumButton/Button.scss

```diff
@@ -26,4 +33,322 @@
+                        background-color: transparent;
+                        color: white;
+                        border-color: white;
+                    }
+                    &:disabled {
+                        background-color: transparent;
+                        color: rgba(white, .5);
+                        border-color: rgba(white, .5);
+                    }
+                }
+                &-LIGHT {
+                    background-color: #0a0a1e;
+                    color: white;
+                    border-color: #0a0a1e;
+                    &:hover {
+                        background-color: transparent;
+                        color: #0a0a1e;
+                        border-color: #0a0a1e;
+                    }
+                    &:not(:disabled):not(.disabled):active {
+                        background-color: transparent;
+                        color: #0a0a1e;
+                        border-color: #0a0a1e;
+                    }
+                    &:disabled {
+                        background-color: transparent;
+                        color: rgba(white, .5);
+                        border-color: rgba(white, .5);
+                    }
+
+                }
+            }
+        }
+        &.btn-secondary {
+            background-color: transparent;
+
+            &.color-scheme {
+                &-DARK {
+                    color: white;
+                    border-color: white;
+                    &:hover {
+                        background-color: white;
+                        color: black;
+                        border-color: white;
+                    }
+                    &:not(:disabled):not(.disabled):active {
+                        background-color: rgba(white, .5);
+                        color: black;
+                        border-color: transparent;
+                    }
+                    &:disabled {
+                        background-color: transparent;
+                        color: rgba(white, .5);
+                        border-color: rgba(white, .5);
+                    }
+                }
+                &-LIGHT {
+                    background-color: white;
+                    color: black;
+                    border-color: white;
+                    &:hover {
+                        color: white;
+                        background-color: #0A0A1E;
+                        border-color: #0A0A1E;
+                    }
+                    &:not(:disabled):not(.disabled):active {
+                        background-color: rgba(10, 10, 30, 0.5);
+                        color: white;
+                        border-color: transparent;
+                    }
+                    &:disabled {
+                        background-color: rgba(#0A0A1E,.5);
+                        color: rgba(#0A0A1E,.5);
+                        border-color: transparent;
+                    }
+                    &:disabled {
```

Comment:
Duplicate code

Reply:
It Will be fixed soon, now it's hard to predict the structure of a mixin